nix/mobile/default.nix

```diff
@@ -7,12 +7,12 @@ let
 
   android = callPackage ./android {
     inherit jsbundle;
-    status-go = status-go.mobile.android;
+    status-go-func = status-go.mobile.android_func;
```

Comment:
I don't think we need any of this renaming. Even before your changes `status-go` was a function, you just nested it into another function, but because it has default arguments it doesn't really matter because at evaluation time it all gets collapsed into that final result. Lets drop this `-func` stuff.